### DISCUSSION TITLE
Topics: editing not possible

### DISCUSSION THREAD

**Discussion by AndreMarschalek** [community/community#21592] [New to GitHub]
<p>Trying to add Topics to a repository like for example <code>c#</code> is not possible because of</p>
<blockquote>
<p>Error saving your changes: Repository topics Topics must start with a lowercase letter or number, consist of 35 characters or less, and can include hyphens.</p>
</blockquote>
<p>what’s wrong here?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<blockquote>
<p>what’s wrong here?</p>
</blockquote>
<p>The error message is pretty bad. I’ll flag this for the team responsible!</p>
<p>For clarity: topics can contain up to 35 of:</p>
<ul>
<li>lowercase letters</li>
<li>numbers</li>
<li>hyphens</li>
</ul>
<p>So you can use <a href="https://github.com/topics/csharp">csharp</a> as your topic, but not c#.</p>

**Reply by Hjhiiik** [reply]
Transfer mony

**Comment by CaseyG5** [comment]
<p>Still getting the same error message.  Not a big deal, but I saw the option to add tags so I figured I’d add a few.  Even adding just one or two short ones with only lowercase letters does not work.  Again, no worries.  I’m sure this issue is very low in priority.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/7/3782335c07abae60bf0ff97da79c978cafe3670c.png" data-download-href="/uploads/short-url/7V3dif08qLUJDm4SMQmvnD9HQrq.png?dl=1" title="Repo topics Topics error" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/16518948/181065754-2e983700-d583-452d-b8c0-59e332b8990d.png" alt="Repo topics Topics error" data-base62-sha1="7V3dif08qLUJDm4SMQmvnD9HQrq" width="690" height="87" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/7/3782335c07abae60bf0ff97da79c978cafe3670c_2_690x87.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/7/3782335c07abae60bf0ff97da79c978cafe3670c_2_1035x130.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/3/7/3782335c07abae60bf0ff97da79c978cafe3670c.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/7/3782335c07abae60bf0ff97da79c978cafe3670c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Repo topics Topics error</span><span class="informations">1035×131 13.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
